proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,139 @@
+# MDM Base Setup @ Dogfood
+
+## Items of Note
+- A NO-OP command (setup apple-mdm) which are pure instruction gathering only
+- Setup integrated into fleetctl, not namespaced into a separate vertical (e.g. `fleetctl apple-mdm`). That might happen at other parts of the UX, but not for setup.
+
+## fleetctl setup
+
+```
+$ fleetctl setup
+NAME:
+   fleetctl setup - Set up a Fleet instance or MDM
+
+USAGE:
+   fleetctl setup [options]
+   fleetctl setup apple-mdm
+   fleetctl setup apns
+   fleetctl setup scep
+   
+SUBCOMMANDS:
+  apple-mdm  Instructions for enabling MDM for MacOS devices
+  apns       Apple Push Notification Service to enable MDM for MacOS devices
+  scep       Simple Certificate Enrollment Protocol to enable MDM for MacOS devices
+
+OPTIONS:
+	... // remains same
+```
+
+```
+$ fleetctl setup apple-mdm
+NAME:
+  fleetctl setup apple-mdm - Instructions for enabling MDM for MacOS devices
+
+INSTRUCTIONS:
+  Note: Apple requires Fleet to verify contact info for MDM cert users. We intend to automate this process soon, but for now, if you are interested in trying out MacOS MDM via Fleet, please schedule a quick friendly 15-min with us to get set up: https://calendly.com/fleetmo/fleet-mdm-intro-and-setup.
```

Comment:
"Apple requires Fleet to verify contact info for MDM cert users"

The certificate in question is called "APNs certificate for MDM" (from https://developer.apple.com/business/documentation/MDM-Protocol-Reference.pdf).